framework/integration/test/reconciliation/reconciliation_test.go

```diff
@@ -3,19 +3,113 @@
 package reconciliation
 
 import (
+	"reflect"
 	"testing"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/giantswarm/operatorkit/framework/integration/client"
 )
 
 // Test_Finalizer_Integration_Reconciliation is a integration test for
 // the proper replay and reconciliation of delete events with finalizers.
 func Test_Finalizer_Integration_Reconciliation(t *testing.T) {
+	configMapName := "test-cm"
+	testFinalizer := "operatorkit.giantswarm.io/test-operator"
 	testNamespace := "finalizer-integration-reconciliation-test"
+	testOtherFinalizer := "operatorkit.giantswarm.io/other-operator"
+	operatorName := "test-operator"
 
 	client.MustSetup(testNamespace)
 	defer client.MustTeardown(testNamespace)
 
-	// TODO: Implement the actual test here.
+	operatorkitFramework, err := client.NewFramework(operatorName, testNamespace)
+	if err != nil {
+		t.Fatal("expected", nil, "got", err)
+	}
+
+	// We create a configmap, but add a finalizer of another operator. This will
+	// cause the configmap to continue existing after the framework removes it own
+	// finalizer.
+	cm := &corev1.ConfigMap{
```

Comment:
Ack, I'll take a step back and create another PR then and later merge here because there are going to be quite a few changes to test with a CR instead.

Reply:
That's a valid point. Maybe we can do a parallel tests one with CR and second with core object.